package.json

```diff
@@ -247,8 +249,9 @@
   },
   "lint-staged": {
     "*.{js,ts,tsx}": [
-      "yarn lint:fix",
-      "yarn prettier"
+      "bash -c \"yarn typecheck\"",
```

Comment:
Necessary instead of just `yarn typecheck` because `tsc` loses its configuration if ran by `lint-staged`